build/webpack.config.js

```diff
@@ -27,7 +27,7 @@ const webpackConfig = {
 
 const webpackHotPath = `${config.compiler_public_path}__webpack_hmr`
 
-export const webpackHotMiddlewareEntry = `webpack-hot-middleware/client?${_.map({
+const webpackHotMiddlewareEntry = `webpack-hot-middleware/client?${_.map({
```

Comment:
This export was never used.  I searched the code base as well as tested all the build scripts.  All is well.